chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  a["networks"]["ironic"] = ta["networks"]["ironic"]
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
+  end
```

Comment:
This is similar here, I'm afraid that intf3 could exist already and that you could be overriding it...